testpar/API/H5_api_test_parallel.c

```diff
@@ -186,17 +186,102 @@ parse_command_line(int argc, char **argv)
 }
 
 static void
-usage(void)
+usage(FILE *stream)
 {
-    print_func("file        run only the file interface tests\n");
-    print_func("group       run only the group interface tests\n");
-    print_func("dataset     run only the dataset interface tests\n");
-    print_func("attribute   run only the attribute interface tests\n");
-    print_func("datatype    run only the datatype interface tests\n");
-    print_func("link        run only the link interface tests\n");
-    print_func("object      run only the object interface tests\n");
-    print_func("misc        run only the miscellaneous tests\n");
-    print_func("async       run only the async interface tests\n");
+    fprintf(stream, "file        run only the file interface tests\n");
+    fprintf(stream, "group       run only the group interface tests\n");
+    fprintf(stream, "dataset     run only the dataset interface tests\n");
+    fprintf(stream, "attribute   run only the attribute interface tests\n");
+    fprintf(stream, "datatype    run only the datatype interface tests\n");
+    fprintf(stream, "link        run only the link interface tests\n");
+    fprintf(stream, "object      run only the object interface tests\n");
+    fprintf(stream, "misc        run only the miscellaneous tests\n");
+    fprintf(stream, "async       run only the async interface tests\n");
+}
+
+static int
+create_test_container(char *filename, uint64_t vol_cap_flags)
```

Comment:
When `create_test_container` was changed to `H5_api_test_create_containers` for the serial API tests, the occurrence in the parallel API tests wasn't changed, so the parallel tests don't currently build in this branch. I've replicated the old `create_test_container` here for now so they will build, but this should be resolved. `H5_api_test_create_containers` will need to be moved to a more accessible scope and `H5_api_test_destroy_container_files` will need to be made more general to be used in the parallel tests.